modules/rosa-delete-users.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Module included in the following assemblies:
+//
+// getting_started_rosa/rosa-creating-cluster.adoc
+
+
+[id="rosa-delete-users"]
+= Revoking administrator access using the OCM console
+You can revoke the `dedicated-admin` or `cluster-admin` access of users through the OpenShift Cluster Manager (OCM) console. Users will be able to access the cluster without administrator privileges.
+
+.Prerequisites
+
+* You have added an Identity Provider (IDP) to your cluster.
+* You have the IDP user name for the user whose privileges you are revoking.
+* You are logged in to the cluster as the user who created the cluster, the organization administrator user, or the superadministator user.
```

Comment:
It should log in to OCM console rather than 'logged in to the cluster'. And It'e better use 'OCM account' rather than 'user' here to log in the OCM console.

Reply:
Incorporated this feedback.